core/pom.xml

```diff
@@ -319,6 +319,12 @@
       <artifactId>selenium-java</artifactId>
       <scope>test</scope>
     </dependency>
+    <!-- Added for selenium: -->
+    <dependency>
+      <groupId>xml-apis</groupId>
+      <artifactId>xml-apis</artifactId>
+      <scope>test</scope>
```

Comment:
It does, but for some reason, the `org.w3c` class I mentioned doesn't turn up at runtime during the tests. It doesn't happen in a plain vanilla build, but does with the Hadoop 2-style build I ran. The class in question appears to only come from `xml-apis`.

This looks like the key difference in `dependency:tree` between the two:

```
192,193c333
< [INFO] |  |  |  +- xerces:xercesImpl:jar:2.11.0:test
< [INFO] |  |  |  |  \- xml-apis:xml-apis:jar:1.4.01:test
---
> [INFO] |  |  |  +- (xerces:xercesImpl:jar:2.11.0:test - omitted for conflict with 2.9.1)
```

(Hadoop 2 on the right / `>`) Looks like something trumps Xerces 2.11 in test scope, with 2.9.1, and that brings in a different xml-apis that doesn't work.

I specified `tests` in the parent to make sure we don't inadvertently manage non-test deps, and here for clarity.